Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml

```diff
@@ -164,7 +164,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-context</artifactId>
-      <version>${dependency.spring.framework.version}</version>
+      <version>${spring.version}</version>
```

Comment:
Isn't this dep on the parent dep management?

Reply:
yes, agree